docs/github/builder.md

```diff
@@ -286,6 +286,23 @@ Parameters without a value will be considered booleans (with a value of true).
 _**required:** `false`_
 _**default:** `""`_
 
+## Outputs
+
+Below are outputs that can be accessed by using `${{ steps.UNITY_BUILDER_STEP_ID.outputs.OUTPUT_NAME }}`, where
+`UNITY_BUILDER_STEP_ID` is the id of the Builder step, and `OUTPUT_NAME` is the name of the output.
```

Comment:
I think it's better to explain the id and output name by using camelCase variables like `buildStep` and `outputName` and then also use `buildStep` as the id in the examples.